src/main/java/com/atlassian/db/replica/api/ThrottledCache.java

```diff
@@ -23,6 +23,11 @@
         return Optional.ofNullable(value);
     }
 
+    @Override
+    public Optional<T> get() {
+        return Optional.ofNullable(value);
```

Comment:
When is it ok to skip `maybeRefresh`?
The `Supplier` isn't executed if the lock is closed.

Reply:
It's always ok to read the old value from the cache.

Reply:
Hmm, that's true for `SuppliedCache`, it never guaranteed any degree of freshness.